ExampleApps/SwiftUIExample/SwiftUIExampleTests/Views/TermsAndConditionsTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  TermsAndConditionsTests.swift
+//  SwiftCurrent
+//
+//  Created by Richard Gist on 10/1/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//  swiftlint:disable multiline_function_chains
+
+import XCTest
+import SwiftUI
+import ViewInspector
+
+@testable import SwiftCurrent_SwiftUI // 🤮 it sucks that this is necessary
+@testable import SwiftUIExample
+
+final class TermsAndConditionsTests: XCTestCase, View {
+    func testLayout() {
+        let exp = ViewHosting.loadView(TermsAndConditions()).inspection.inspect { view in
+            XCTAssertEqual(view.findAll(ViewType.Button.self).count, 2)
+        }
+        wait(for: [exp], timeout: TestConstant.timeout)
+    }
+
+    func testPrimaryAcceptButtonCompletesWorkflow() {
+        let workflowFinished = expectation(description: "View Proceeded")
+        let exp = ViewHosting.loadView(WorkflowLauncher(isLaunched: .constant(true)) {
+            thenProceed(with: TermsAndConditions.self)
+        }.onAbandon {
+            XCTFail("Abandon should not have been called")
```

Comment:
Was this part of a refactor? I generally am wary of testing things that didn't happen, cause where do you stop?